packages/sawja/sawja.1.5.10/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "sawja@inria.fr"
+authors: "Sawja development team"
+homepage: "http://sawja.inria.fr"
+bug-reports: "https://github.com/javalib-team/sawja/issues"
+license: "GPL-3.0"
+dev-repo: "git+https://github.com/javalib-team/sawja.git"
+build: [
+  ["./configure.sh"]
+  [make]
+]
+remove: [
```

Comment:
This remove stanza is no longer needed since opam 2.0. Opam tracks the installed packages automatically now